File: VB/WindowsApplication1/Minutes1Scale.vb
' Developer Express Code Central Example:
' How to resize an appointment without snapping it to time cells
' 
' This example illustrates how to resize an appointment without snapping it to
' timecells.
' 
' You can find sample updates and versions for different programming languages here:
' http://www.devexpress.com/example=E2341

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraScheduler

Namespace Lane_Management
    Public Class Minutes1Scale
        Inherits LessThanADayTimeScale

        Private Const myScaleValue As Integer = 1
        Public Sub New()
            MyBase.New(myScaleValue)
        End Sub
        Public Sub New(ByVal ScaleValue As Integer)
            MyBase.New(ScaleValue)

        End Sub


        Protected Overrides ReadOnly Property DefaultDisplayFormat() As String
            Get
                Return "h:mm tt"
            End Get
            'get { return "HH:mm"; }
        End Property

        Protected Overrides ReadOnly Property DefaultMenuCaption() As String
            Get
                Return "1Minutes"
            End Get
        End Property

    End Class
End Namespace

File: CS/WindowsApplication1/Minutes1Scale.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraScheduler;

namespace Lane_Management
{
    public class Minutes1Scale : LessThanADayTimeScale
    {
        const int myScaleValue = 1;
        public Minutes1Scale()
            : base(myScaleValue)
        {
        }
        public Minutes1Scale(int ScaleValue)
            : base(ScaleValue)
        {
            
        }
         

        protected override string DefaultDisplayFormat
        {
            get { return "h:mm tt"; }
            //get { return "HH:mm"; }
        }

        protected override string DefaultMenuCaption
        {
            get
            {
                return "1Minutes";
            }
        }

    }
}
